addons/point_of_sale/static/tests/pos/tours/ticket_screen_tour.js

```diff
@@ -283,3 +283,37 @@ registry.category("web_tour.tours").add("LotTour", {
             }),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("OrderTimeTour", {
+    checkDelay: 50,
+    steps: () =>
+        [
+            ProductScreen.setTimeZone("Asia/Tokyo"),
+            Chrome.startPoS(),
+            Dialog.confirm("Open Register"),
+
+            ProductScreen.clickDisplayedProduct("Desk Pad"),
+
+            Chrome.clickOrders(),
+            {
+                content: "Validate order time matches local timezone",
+                trigger: ".orders .order-row:first .small.text-muted",
+                run: function () {
+                    const displayedTimeElement = document.querySelector(
+                        ".orders .order-row:first-child .small.text-muted"
+                    );
```

Comment:
```suggestion
                run: function ({ anchor: displayedTimeElement }) {
```

The specified `run` function is provided a `TourHelpers` instance with `anchor` props pointing to the element selected by the `trigger`.

Reply:
Hello @caburj , its updated